stable/insights-agent/templates/rbac-reporter/_job.yaml

```diff
@@ -0,0 +1,41 @@
+{{- define "rbacreporter.job.spec" -}}
```

Comment:
We probably don't have to keep up defining the separate _job file since we don't have both CronJob and Job anymore.

Reply:
Not sure i understand this one here. You mean `_job.yaml` and `cronjob.yaml` are not needed in this context?

Reply:
Before, we used `_job.yaml` to factor out the common parts of the Job and CronJob. Now we're not creating separate Jobs (just the CronJob) so we don't really need _job.yaml anymore.

I'm good leaving this in for consistency, but we should refactor the chart to get rid of this pattern in the future.